File: Patterns/2. Rectangle Patterns/16. Rectangle Even Numbers/ren.py
l = int(input("Enter the Length : "))
b = int(input("Enter the Breadth : "))

for i in range(1,l+1):
    for j in range(1,b+1):
            print(2*j, end=' ')
    print()

'''
Output:

2 4 6 8 10 12 14 
2 4 6 8 10 12 14 
2 4 6 8 10 12 14 
2 4 6 8 10 12 14 
2 4 6 8 10 12 14   
'''

File: Patterns/2. Rectangle Patterns/16. Rectangle Even Numbers/ren.cpp
#include <iostream>
using namespace std;
int main(){
    int l,b;
    cout<<"Enter the Length : ";
    cin>>l;
    cout<<"Enter the Breadth : ";
    cin>>b;

    for(int i=1; i<=l; i++){
        for(int j=1; j<=b; j++){
            cout<<2*j<<" ";
        }
        cout<<'\n';
    }
}

/*
Output

2 4 6 8 10 12 14 
2 4 6 8 10 12 14 
2 4 6 8 10 12 14 
2 4 6 8 10 12 14 
2 4 6 8 10 12 14 

*/